core/src/main/scala/org/apache/spark/util/JsonProtocol.scala

```diff
@@ -32,6 +32,7 @@ import org.apache.spark.executor._
 import org.apache.spark.scheduler._
 import org.apache.spark.storage._
 import org.apache.spark._
+import org.apache.hadoop.hdfs.web.JsonUtil
```

Comment:
@JoshRosen you're right. Is this something you're looking after or do you want me to raise a PR? Should I make a JIRA? Its breaking my mapr3 build